### ISSUE TITLE
Enhance ARIA roles and attributes for the components

### ISSUE THREAD

**Issue by yigitfindikli** [primefaces/primeng#8439]
Review and apply required aria roles and attributes for the components for better accessibility compliance.

**Comment by kkdattaGIT** [comment]
@yigitfindikli / @cagataycivici  wondering if you have looked at the issue raised by SelenaTuyen?
We have critical bugs for Accessibility Tests failing for the limitation mentioned above.
